charmed_openstack_info/data/lp-builder-config/ceph.yaml

```diff
@@ -4,11 +4,10 @@ defaults:
   branches:
     master:
       build-channels:
-        charmcraft: "2.x/stable"
+        charmcraft: "2.2/stable"
       channels:
-        - latest/edge
+        - reef/candidate
```

Comment:
I think you probably want to *keep* `latest/edge`, and so this probably ought to look like:

```yaml
channels:
  - latest/edge
  - reef/candidate
```

Reply:
Thank you, you're absolutely right